UI/invite-card.tsx

```diff
@@ -33,6 +36,13 @@ export function InviteCard(props: {
     };
 
     const profile = profileGetter.getProfilesByPublicKey(publicKey);
+    const open = () => {
+        props.emit({
+            type: "SelectConversation",
+            pubkey: publicKey,
+            isGroupChat: true,
+        });
+    };
```

Comment:
Not sure if you are aware of this, but you can write this function outside of local scope